bcl/src/bc_module_rs485.c

```diff
@@ -0,0 +1,403 @@
+#include <bc_module_rs485.h>
+
+
+#define _BC_MODULE_RS485_I2C_UART_ADDRESS 0x4e
+#define _BC_MODULE_RS485_I2C_TLA2021_ADDRESS 0x48
+
+#define _BC_SC16IS7x0_REG_IER       0x01 << 3
```

Comment:
Always use `(...)` for compound `#define` statements, also we use all lowercase in hex numbers